internal/routes/routes.go

```
@@ -8,84 +8,104 @@ import (
 	bot.Handle("/start", resolver.DefaultHandler.Start)
 	bot.Handle("/token", middleware.RequireRegistration(resolver.DefaultHandler.GetToken, app))
 
 	bot.Handle("/debug", func(context telebot.Context) error {
+		app.Logger.Info(op+".", context, "Debug command triggered")
 		return handlers.DebugMessage(context, app)
 	})
-
 }
 
 func SetupMovieRoutes(bot *telebot.Bot, container *api.Resolver, app *appCfg.App) {
+	const op = "routes.SetupMovieRoutes"
```

golangci-lint (1.64.8)

const `op` is unused

(unused)